bigbluebutton-html5/private/config/settings.yml

```diff
@@ -41,6 +41,9 @@ public:
       dataSaving:
         viewParticipantsWebcams: true
         viewScreenshare: true
+      webRTCStats:
+        logging: true
+        interval: 200
```

Comment:
Is such a short interval needed? Would logging every second not be sufficient or is there a need to have it be more frequent?